### ISSUE TITLE
Supplying address of the repo as one argument.

### ISSUE THREAD

**Issue by lehins** [tfausak/github-release#3]
It would nice to be able to supply an argument that can take repo name an owner as one argument, instead of two (`--owner` and `--repo`), which would be useful for CIs like AppVeyor and Travis, since they have environment variables `APPVEYOR_REPO_NAME="name/repo-name"` and `TRAVIS_REPO_SLUG="owner_name/repo_name"` respectively.

In case when owner and repo names are available as separate env variables, while writing a script on any other CI or elsewhere, it is still easier to combine them rather than it is to split them.

**Comment by tfausak** [comment]
What would be a good way to handle this in the CLI interface? This is what I'm thinking:

``` sh
> github-release upload --owner foo --repo bar ... # currently
> github-release upload --repo foo/bar ... # --owner foo --repo bar
> github-release upload --owner ignored --repo foo/bar ... # --owner foo --repo bar
> github-release upload --repo foo/bar/extra ... # --owner foo --repo bar/extra
```

In other words, if the repo contains a forward slash, split on the first forward slash and use the first part as the owner. If both an owner and a qualified repo are given, ignore the owner.

**Comment by lehins** [comment]
IMHO having `--owner` flag is redundant, so it would probably make sense to deprecate it, but in a smart backwards compatible way, e.g.:
* `--owner=tfausak` & `--repo=github-release` -> (condition: owner & no slash in repo) print deprecation message.
* `--owner=tfausak` & `--repo=tfausak/github-release` -> (condition: owner supplied and slash in repo) print deprecation message and validate the prefix matches, possibly throw an error if it doesn't, but at least avoid adding the prefix to it again.
* `--repo=tfausak/github-release` -> we have all the info we need, safe to proceed.

Repository names don't have forward slashes in them, so it doesn't look like that going this route would break anything.

**Comment by tfausak** [comment]
Released in [version 1.1.0](https://github.com/tfausak/github-release/releases/tag/1.1.0).
